### PR TITLE
ALF Automation

### PR DESCRIPTION
**Pull request by walfgithub**
Brackets: Please merge this pull request created by ALF. @ArturOledzki @pantkowiak @ybayer FYI

### CODE REVIEW

File: src/nls/fr/strings.js [hunk 1]
```diff
@@ -581,7 +581,7 @@ define({
 	"CMD_LOG_NODE_STATE": "Noter l’état du nœud dans la console",
 	"CMD_RESTART_NODE": "Redémarrer le nœud",
 	"CMD_SHOW_ERRORS_IN_STATUS_BAR": "Afficher les erreurs dans la barre d’état",
-	"CMD_OPEN_BRACKETS_SOURCE": "Ouvrir la source Brackets",
+	"CMD_OPEN_BRACKETS_SOURCE": "[7693766] !é=Open {APP_NAME} Source=!",
```
Comment (prafulVaishnav) [inline src/nls/fr/strings.js]: Looking weird..

File: src/nls/fr/strings.js [hunk 2]
```diff
@@ -610,7 +610,7 @@ define({
...
+	"DETECTED_EXCLUSION_INFO": "[7629443] !é={APP_NAME} ran into trouble processing <span class='dialog-filename'>{0}</span>.<br><br>This file will no longer be processed for code hints, Jump to Definition or Quick Edit. To reenable this file, open <code>.brackets.json</code> in your project and edit <code>jscodehints.detectedExclusions</code>.<br><br>This is likely a {APP_NAME} bug. If you can provide a copy of this file, please <a href='https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue'>file a bug</a> with a link to the file named here.=!",
```
Comment (prafulVaishnav) [inline src/nls/fr/strings.js]: Looking weird..

**Comment by prafulVaishnav** [comment]
@walfgithub The conversion is looing weird.. Can you take a look..

**Comment by abose** [comment]
@walfgithub can you give us more context on what this change is about?

**Comment by ApfelUser** [comment]
@prafulVaishnav @abose I am not really sure but I think @walfgithub is some kind of bot created by Adobe

**Comment by prafulVaishnav** [comment]
@abose This change is due to [this](https://github.com/adobe/brackets/pull/10800) PR

**Comment by abose** [comment]
Ok, got it.  the change looks odd.

**Comment by redmunds** [comment]
String that look like `"[7693766] !é=<English version of string>=!"` are temporary placeholders used in the localization process. These are used to mark that a string still needs to be localized and also provides a longer string (because many languages strings are longer than English) to verify UI is not broken.

Another commit will be coming from WALF soon with final translations.

Be sure that these are all fixed before final release.

**Comment by abose** [comment]
Thanks @redmunds . The new transalations are submitted now, so merging.

**Pull request merged**